### ISSUE TITLE
Adding new entities in config file disable all the mod

### ISSUE THREAD

**Issue by PabloDnSS** [Tschipcraft/spawnanimations#24]
### Describe the bug

Maybe im doing something wrong i don't know but as soon i put one custom entities the mod is not working

{
  "//comment": "Entities specified here will dig themselves up when a player is near or other exclusion criteria are met. You can add your own entities here",
  "replace": false,
  "values": [
    "minecraft:zombie",
    "minecraft:husk",
    "minecraft:drowned",
    "minecraft:skeleton",
    "minecraft:stray",
    "minecraft:spider",
    "minecraft:cave_spider",
    "minecraft:creeper",
    "minecraft:wither_skeleton",
    "minecraft:hoglin",
    "minecraft:zombified_piglin",
    "minecraft:strider",
    "minecraft:enderman",
    "minecraft:giant",
    "minecraft:magma_cube",
    "minecraft:slime"
    "quark:forgotten",
    "quark:wraith",
    "alexsmobs:mimicube",
    "alexsmobs:cockroach",
    "alexsmobs:centipede",
    "alexsmobs:tarantula_hawk",
    "alexsmobs:underminer",
    "aquamirae:tortured_soul",
    "creeperoverhaul:jungle_creeper",
    "creeperoverhaul:bamboo_creeper",
    "creeperoverhaul:desert_creeper",
    "creeperoverhaul:badlands_creeper",
    "creeperoverhaul:hills_creeper",
    "creeperoverhaul:savannah_creeper",
    "creeperoverhaul:mushroom_creeper",
    "creeperoverhaul:swamp_creeper",
    "creeperoverhaul:dripstone_creeper",
    "creeperoverhaul:cave_creeper",
    "creeperoverhaul:dark_oak_creeper",
    "creeperoverhaul:spruce_creeper",
    "creeperoverhaul:beach_creeper",
    "creeperoverhaul:snowy_creeper",
    "creeperoverhaul:ocean_creeper",
    "graveyard:skeleton_creeper",
    "graveyard:reaper",
    "graveyard:revenant",
    "graveyard:nightmare",
    "graveyard:corrupted_vindicator",
    "graveyard:corrupted_pillager",
    "graveyard:corrupted_champion",
    "graveyard:acolyte",
    "endermanoverhaul:cave_enderman", 
    "endermanoverhaul:badlands_enderman",
    "endermanoverhaul:crimson_forest_enderman",
    "endermanoverhaul:dark_oak_enderman",
    "endermanoverhaul:desert_enderman",
    "endermanoverhaul:end_enderman",
    "endermanoverhaul:end_islands_enderman",
    "endermanoverhaul:flower_fields_enderman",
    "endermanoverhaul:ice_spikes_enderman",
    "endermanoverhaul:mushroom_fields_enderman",
    "endermanoverhaul:nether_wastes_enderman",
    "endermanoverhaul:coral_enderman",
    "endermanoverhaul:savanna_enderman",
    "endermanoverhaul:snowy_enderman",
    "endermanoverhaul:soulsand_valley_enderman",
    "endermanoverhaul:swamp_enderman",
    "endermanoverhaul:warped_forest_enderman",
    "endermanoverhaul:windswept_hills_enderman",
    "endermanoverhaul:scarab_enderman",
    "endermanoverhaul:spirit_enderman",
    "illagerinvasion:surrendered",
    "realmrpg_creep:creepling",
    "realmrpg_creep:halloweed",
    "alexscaves:watcher",
    "alexscaves:forsaken",
    "born_in_chaos_v1:decrepit_skeleton",
    "born_in_chaos_v1:skeleton_demoman",
    "born_in_chaos_v1:decaying_zombie",
    "born_in_chaos_v1:baby_skeleton",
    "born_in_chaos_v1:siamese_skeletons",
    "born_in_chaos_v1:barrel_zombie",
    "born_in_chaos_v1:phantom_creeper",
    "born_in_chaos_v1:door_knight",
    "born_in_chaos_v1:restless_spirit",
    "born_in_chaos_v1:skeleton_trasher",
    "born_in_chaos_v1:pumpkin_spirit",
    "born_in_chaos_v1:seared_spirit", 
    "born_in_chaos_v1:firelight",
    "born_in_chaos_v1:diamond_termite",
    "born_in_chaos_v1:bonescaller",
    "born_in_chaos_v1:supreme_bonescaller",
    "born_in_chaos_v1:bone_imp",
    "born_in_chaos_v1:nightmare_stalker",
    "born_in_chaos_v1:dark_vortex",
    "born_in_chaos_v1:dread_hound",
    "born_in_chaos_v1:dire_hound_leader",
    "born_in_chaos_v1:fallen_chaos_knight",
    "born_in_chaos_v1:scarlet_persecutor",
    "born_in_chaos_v1:spirit_guide",
    "born_in_chaos_v1:zombie_clown",
    "born_in_chaos_v1:swarmer",
    "born_in_chaos_v1:zombie_fisherman",
    "born_in_chaos_v1:zombie_lumberjack",
    "born_in_chaos_v1:zombie_bruiser",
    "born_in_chaos_v1:missioner",
    "born_in_chaos_v1:senor_pumpkin",
    "born_in_chaos_v1:mr_pumpkin",
    "born_in_chaos_v1:sir_pumpkinhead",
    "born_in_chaos_v1:lord_pumpkinhead"
  ]
}

**Comment by PabloDnSS** [comment]
ty for the fast reply 
[latest.log](https://github.com/Tschipcraft/spawnanimations/files/14410211/latest.log)

**Comment by Tschipcraft** [comment]
Are you sure that you moved the mod back into the mods folder? There is no single mention about spawnanimations in the log

**Comment by PabloDnSS** [comment]
my bad since i had no luck with the mod i tried with the datapack 

here is the log with the mod

[latest.log](https://github.com/Tschipcraft/spawnanimations/files/14417622/latest.log)

**Comment by Tschipcraft** [comment]
Oh, and you'd need to join a world since Spawn Animations doesn't fully initialize itself until then 😅 Nothing out of the ordinary just yet..

**Comment by PabloDnSS** [comment]
Yep fresh world everytime just tried again with the mod and nothing and tried with both datapack and mod nothing

**Comment by PabloDnSS** [comment]
must be something with all the mods i have cause i just tried to config visuality mod and the same is happening

**Comment by Tschipcraft** [comment]
No, I meant join a world and then send the log. The most recent one ends before a world loads.

**Comment by PabloDnSS** [comment]
[latest.log](https://github.com/Tschipcraft/spawnanimations/files/14421861/latest.log)

**Comment by Tschipcraft** [comment]
Thanks! I found the cause, you forgot to add a comma after `minecraft:slime`. Not sure why the GitHub syntax highlighting didn't show that.

**Comment by PabloDnSS** [comment]
ok thank you and sry for the trouble

**Comment by PabloDnSS** [comment]
[latest.log](https://github.com/Tschipcraft/spawnanimations/files/14423375/latest.log)
 still don't work

**Comment by Tschipcraft** [comment]
`graveyard:corrupted_champion`, `endermanoverhaul:scarab_enderman`, `endermanoverhaul:spirit_enderman` and `born_in_chaos_v1:skeleton_trasher` are invalid entries apparently, try removing or correcting them

**Comment by PabloDnSS** [comment]
Ok perfect it's working so the problem is that the mod can't render too big models or too little(like a pixie) thank you

**Comment by Tschipcraft** [comment]
Interesting, that shouldn't actually prevent the mobs to be accepted in the tag list.. I'll do some testing with the invalid entries when I get around to do so

**Comment by PabloDnSS** [comment]
One more little thing is there a way to deal with the glowing enderman eyes?

**Comment by Tschipcraft** [comment]
Sorry for missing the last comment, this is the same issue as #25. You can read my response about it there
